gpu/kinfu_large_scale/tools/kinfuLS_app.cpp

```diff
@@ -889,7 +891,8 @@ struct KinFuLSApp
     
   }
 
-  void source_cb1(const boost::shared_ptr<openni_wrapper::DepthImage>& depth_wrapper)  
+  template<typename DepthImg>
```

Comment:
Can we constraint the type or provide a default one who's interface is used to make it easier to understand the design?

Reply:
Note that this is a local function in an implementation file; not a part of PCL API. As such, it does not need to be well-documented and easy to understand.